### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by mladjan-gadzic**

### CODE REVIEW

File: trino-hms-hdfs-ranger/setup/build_projects.sh [hunk 1]
```diff
@@ -105,6 +105,18 @@ if [ "$buildRanger" == 0 ]; then
     fi
     exit 1
   fi
+
```
Comment (xBis7) [inline trino-hms-hdfs-ranger/setup/build_projects.sh:108]: Can you add right after every java export this?

```
  export MAVEN_OPTS="-Xss64m -Xmx2g -XX:ReservedCodeCacheSize=1g"
```

**Review by xBis7 — APPROVED** [review]

File: trino-hms-hdfs-ranger/setup/build_projects.sh (hunk 1 above)
Comment (mladjan-gadzic) [reply trino-hms-hdfs-ranger/setup/build_projects.sh:108]: 2g didn't work for me. Can I put it to 4g instead?

Comment (xBis7) [reply trino-hms-hdfs-ranger/setup/build_projects.sh:109]: If 2 isn't enough, let's move to 4. I guess it depends on the machine.

File: trino-hms-hdfs-ranger/setup/build_projects.sh [hunk 2]
```diff
@@ -105,6 +107,20 @@ if [ "$buildRanger" == 0 ]; then
...
+  if [ "$ranger_image" == "true" ]; then
```
Comment (xBis7) [inline trino-hms-hdfs-ranger/setup/build_projects.sh:111]: @mladjan-gadzic Shouldn't the `if` be closed? I think you are missing an `fi`

Comment (mladjan-gadzic) [reply trino-hms-hdfs-ranger/setup/build_projects.sh:111]: Fixed.

File: trino-hms-hdfs-ranger/README.md [hunk 3]
```diff
@@ -4,3 +4,124 @@ Almost each script depends on another. For that reason, the scripts assume that
 
 Check each individual script for parameters and more info.
 
+## Basic Instructions
```
Comment (xBis7) [inline trino-hms-hdfs-ranger/README.md:7]: @mladjan-gadzic can you remove the README changes?

Comment (mladjan-gadzic) [reply trino-hms-hdfs-ranger/README.md:7]: It must be an issue during rebase. Let me fix this

Comment (mladjan-gadzic) [reply trino-hms-hdfs-ranger/README.md]: Done.

**Pull request merged**